lair_of_xaldern/main.py

```
@@ -15,7 +16,7 @@
 # normalize filename casing
 def clear_screen():
     """Clears the terminal screen."""
-    os.system("cls" if os.name == "nt" else "clear")
+    subprocess.run("cls" if os.name == "nt" else "clear")
```

Ruff (0.15.1)

[error] `subprocess` call: check for execution of untrusted input

(S603)